src/map/ai/controllers/trust_controller.cpp

```diff
@@ -488,7 +488,7 @@ uint8 CTrustController::GetPartyPosition()
     TracyZoneScoped;
 
     auto& trustList = static_cast<CCharEntity*>(POwner->PMaster)->PTrusts;
-    for (std::size_t i = 0; i < trustList.size(); ++i)
+    for (uint8 i = 0; i < trustList.size(); ++i)
```

Comment:
LGTM might complain about this, since trustList.size() might return size_t

Reply:
Huh, I was on autopilot when I changed that, I can't see why it would have thrown a warning